src/styles/product-card.css

```diff
@@ -0,0 +1,94 @@
+.product-card {
+  width: 200px;
+  padding: 33px 17px 16px;
+
+  --main-text-color: #060b35;
+  --product-code-color: #616070;
+  --button-color: #00acdc;
+  --button-text-color: #fff;
+}
+
+.product-card__image {
+  display: flex;
+  justify-content: center;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.product-card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main-text-color);
+  margin-bottom: 4px;
+}
+
+.product-card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--product-code-color);
+  margin-bottom: 16px;
+}
+
+.product-card___statistic {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 23px;
+}
+
+.product-card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--main-text-color);
+}
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  align-items: center;
+}
+
+.product-card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
+}
+
+.product-card__cost {
+  font-weight: 700;
+  font-size: 16px;
+  color: var(--main-text-color);
+}
+
+.product-card__button {
+  font-weight: 700;
+  font-size: 14px;
+}
+
+.product-card__link {
+  height: 40px;
+  width: 166px;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  text-decoration: none;
+  text-transform: uppercase;
+
+  color: var(--button-text-color);
+  background: var(--button-color);
+  border-radius: 5px;
+  transition: 0.2s;
+}
+
+.product-card__link:hover {
+  color: var(--button-color);
+  background: var(--button-text-color);
+  border: var(--button-color) 1px solid;
+  transition: 0.2s;
```

Comment:
specify the transition properties, it is set `transition: all` by default 😉

Reply:
Done!